packages/design-system/src/components/enhanced-tooltip.tsx

```diff
@@ -117,12 +111,6 @@ export const EnhancedTooltip = forwardRef(
         handleFocusEventsDebounced("blur");
         event.preventDefault();
       },
-      onPointerDown: () => {
-        handleDisableOpen();
-      },
-      onKeyDown: () => {
-        handleDisableOpen();
-      },
```

Comment:
@kof @istarkov looks like the tooltips are intentionally disabled once the icon is interacted on inputs. Removed them and everything works as expected. Can you think of any specific case why this is added or it was just a bug ? 🤔

Reply:
is it meant to be only during interaction? to avoid having both tooltip and interaction at the same time?

Reply:
Had the same hunch initially, but even without disabling it. It seems to work without any issues on my side testing.

Reply:
I bet you can hover and then click to drag on the icon and you get the tootlip open at the same time

Reply:
![Screen Recording 2024-01-11 at 23 38 51](https://github.com/webstudio-is/webstudio/assets/11075561/0372de25-4166-4159-b347-1c39556b1bf6)
That's the strage part for me, it wasn't showing the tooltip when the keyboard click happens / drag is in progress. So we don't need to explicitly handle the case here.

Reply:
maybe its for some other case, @istarkov will know

Reply:
Oh I just read the code, its clear why - we enhanced tooltip implements a different delay logic, so it needed to prevent the other events to delay differently